### ISSUE TITLE
TS2416: Property 'add' in type 'Manager<TargetGeometry>' is not assignable to the same property in base type 'IEventManager<TargetGeometry>'.

### ISSUE THREAD

**Issue by grauds** [ddubrava/angular8-yandex-maps#177]
# 🐞 Bug Report

## ✍️ Description

Can't build a project with the latest NX workspace and Angular

## 🔬 Reproduction

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/11165918/194360745-ab83aafb-1b0b-4f67-adf8-d9012c086e12.png">

## 🌎 Environment

  ```
 Node : 16.15.0
   OS   : darwin x64
   npm  : 8.5.5
   
   nx : 14.8.3
   @nrwl/angular : 14.8.3
   @nrwl/cypress : 14.8.3
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.8.3
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : Not Found
   @nrwl/expo : Not Found
   @nrwl/express : 14.8.3
   @nrwl/jest : 14.8.3
   @nrwl/js : 14.8.3
   @nrwl/linter : 14.8.3
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : 14.8.3
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : 14.8.3
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.8.3
   @nrwl/web : 14.8.3
   @nrwl/webpack : 14.8.3
   @nrwl/workspace : 14.8.3
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
         @fortawesome/angular-fontawesome: 0.11.1
         @ng-bootstrap/ng-bootstrap: 13.0.0
```

    "angular8-yandex-maps": "^14.0.2",

**Comment by ddubrava** [comment]
Hey! Thanks for the issue. Unfortunately, I couldn't reproduce it. I also created an Nx workspace with the same dependencies: TypeScript 4.8.4 and Angular 14.2.0. Could you provide a repository with this issue, please.

**Comment by ddubrava** [comment]
Hey! Thanks for the issue. Fixed in the `v14.0.3` release.

**Comment by grauds** [comment]
Many thanks for fixing it, verified, everything is working great now!
